dockers/docker-database/start.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+supervisorctl start redis-server
+
+# If there is a config db dump file, load it
+if [ -r /etc/sonic/config_db.json ]; then
+    # Wait until redis starts
+    while true; do
+        if [ -e /var/run/redis/redis.sock ]; then
+            break
+        fi
+        sleep 1
+    done
```

Comment:
while true; do if [ \"$(redis-cli ping)\" == \"PONG\" ]; then break; fi; sleep 1; done

is more reliable then just check existence of redis.sock